cg/meta/workflow/tower_common.py

```diff
@@ -28,8 +31,16 @@ def get_tower_launch_parameters(cls, tower_pipeline: str, command_args: dict) ->
                     "revision",
                     "compute-env",
                 )
-            ),
+            },
             exclude_true=True,
         )
-        parameters = ["launch"] + tower_options + [tower_pipeline]
-        return parameters
+        return ["launch"] + tower_options + [tower_pipeline]
+
+    @staticmethod
+    def get_tower_id(stdout_lines: Iterable) -> str:
+        """Parse the stdout and return a workflow id."""
+        for line in stdout_lines:
+            if line.strip().startswith("Workflow"):
+                return line.strip().split()[1]
+            else:
+                continue
```

Comment:
Is there not a simpler way to do this? Could you paste an example output?

Reply:
actually the else is not required! I added an example output in the docstring